pyproject.toml

```diff
@@ -31,13 +31,17 @@ dependencies = ["xarray",
     "zarr",
     "fsspec",
     "gcsfs",
-    "pyprojroot"]
+    "pyprojroot",
```

Comment:
Calling `here()` from this pyprojroot seems to limit running this package (dmd-era5) from being run within its own repository (I get an exception with a message like 'project root not found' if I try to run it from elsewhere).

There are alternative ways of getting access to the package data from the source (such as [importlib.resources](https://docs.python.org/3/library/importlib.resources.html#module-importlib.resources)), or perhaps some of the uses of the package root could actually refer to the cwd?

Reply:
@ots22 I saw the use of `here()` in `src/dmd_era5/dvc_tools.py`. For my own learning, please can you explain what `here()` do? I'm still not quite clear after searching online.

Reply:
It was new to me - it seems to be from the package [pyprojroot](https://pypi.org/project/pyprojroot/) and returns the root directory of the project when run from a file inside a subdirectory.

Reply:
> There are alternative ways...

Yes, this sounds like a better approach. I suggest leaving it as is for now, and tackle once we look at #17